transport_analysis/tests/test_velocityautocorr.py

```diff
@@ -371,7 +371,7 @@ def test_self_diffusivity_step_vtraj_all_dims(
         sd_actual = v_simple.self_diffusivity_gk()
         sd_expected = (
             integrate.simpson(
-                characteristic_poly(NSTEP, tdim_factor), range(NSTEP)
+                y=characteristic_poly(NSTEP, tdim_factor), x=range(NSTEP)
```

Comment:
This is an uneducated attempt at a fix (I understand the Python, I don't really use Simpson-based integration regularly enough to know it's doing the right thing).